setup.py

```diff
@@ -61,11 +53,9 @@ def read(path):
       packages=find_packages(),
       namespace_packages=[],
       include_package_data=True,
-      extras_require=dict(
-          test=test_requires,
-      ),
+      extras_require={},
       zip_safe=False,
-      install_requires=requires,
-      tests_require=test_requires,
+      install_requires=[],
```

Comment:
You can omit the keyword arguments if they are "empty":
* extras_require
* install_requires
* tests_equire